hack/export-job-env-vars.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright the KubeVirt Authors.
+#
+#
+
+BASEDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+PROJECT_INFRA_ROOT=$(readlink --canonicalize ${BASEDIR}/..)
+
+function main() {
+    if [ "$#" -eq 0 ]; then
+        usage
+        exit 1
+    fi
+
+    if [[ "$1" =~ -h ]]; then
+        usage
+        exit 0
+    fi
+
+    echo "$(extract_env_vars_to_export "$1")"
+}
+
+function extract_env_vars_to_export() {
+    if [ -z "$1" ]; then
+        usage
+        exit 1
+    fi
+
+    rg -l "$1" "${PROJECT_INFRA_ROOT}/github/ci/prow-deploy/files/jobs" | \
+        xargs yq -r '.periodics[] | select(.name=="periodic-kubevirt-e2e-k8s-1.31-sig-storage-root") | .spec.containers[0].env[] | "export "+.name+"="+.value+ " &&"' | \
```

Comment:
Will this include env variables that are included in presets? You might have to create the prowjob with hack/mkpj.sh to get all env variables.

Reply:
To answer your question - no, it obviously will not include the presets :)